src/pages/dev/express/intro.mdx

```diff
@@ -0,0 +1,76 @@
+
+For protocols choosing to integrate with Axelar Valued Express Executable: 
+1. They need to be whitelisted since the relayer cannot trust arbitrary contracts
+2. The value of the payload in terms of a token is not state or time dependent so that there is no chance the value changes between the express and execute call. This restriction can be removed at the cost of some more gas, and risk to relayer.
+
+## Express ITS Tokens
+For those who are deploying interchain tokens via Axelar's [Interchain Token Service](https://axelar.network/its), those tokens can also be added to the Express Service to provide express transactions for those tokens. ITS makes use of `AxelarValuedExpressExecutable` as the value in an ITS transaction is in the payload.
+
+## Fees
+Express transactions require an additional [fee](https://docs.axelar.dev/dev/reference/pricing#callcontractwithtokenexpress-gmp-express) alongside existing GMP fees. This extra fee acts as insurance for the Express relayer for assuming the risk of an Express transaction.
+{/* TODO confirm $1 with team */}
+
+### Gas Estimation
+When estimating gas for an interchain transaction the [estimateGasFee](https://docs.axelar.dev/dev/axelarjs-sdk/axelar-query-api#estimategasfee) function is available to provide the estimated cost. The same function can be used to estimate gas costs for an express transaction. However, for express you also need to specify several options to specify an accurate gas estimation.
+For example an express transaction going from Polygon to Avalanche;
+```
+await api.estimateGasFee(
+    "Polygon",
+    "Avalanche",
+    "MATIC",
+    500000,
+    1.1,
+    "0",
+    {
+      showDetailedFees: true,
+      transferAmount: inputTokenAmount, // In terms of symbol, not unit denom, e.g. use 1 for 1 axlUSDC, not 1000000
+      sourceContractAddress: sourceChainAddress,
+      destinationContractAddress: destinationChainAddress
+      tokenSymbol: "axlUSDC", //aUSDC for testnet
+    } 
+)
+```
+
+## Risks
+
+### Funding
+While the Express solution is an intuitive design pattern for allowing for fast interchain transactions it is not without any risks. Namely if anything changes on the source chain from the time the transaction is sent to the time it is finalized (due to a [block reorg](https://cointelegraph.com/explained/what-is-chain-reorganization-in-blockchain-technology) for example) and the transaction on the destination chain has already executed then that could lead to lost funds on the source chain as the transaction will have been instantly executed on the destination chain. Due to the potential risk here, funds will need to be provided to onboard to express. Liquidity must be provided by a party to underwrite the risk.
```

Comment:
Find a different wording than "funds will need to be provided to onboard to express". We somewhere need to explain that the relayer is being funded because it can't mint tokens, it only lends them so the transaction can execute quickly, and then must be paid back by the full GMP resolution. This probably deserves its own section.

Reply:
Did not end up providing separate section for this, but I did specifically refer to the relayers "**loaning**" funds which need to be repaid once the regular gmp tx executes on the src.